plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -297,7 +298,7 @@ func NewVolumeZonePredicate(nodeInfo NodeInfo, pvInfo PersistentVolumeInfo, pvcI
 	return c.predicate
 }
 
-func (c *VolumeZoneChecker) predicate(pod *api.Pod, existingPods []*api.Pod, nodeID string) (bool, error) {
+func (c *VolumeZoneChecker) predicate(pod *api.Pod, nodeID string, nodeInfo *schedulercache.NodeInfo) (bool, error) {
```

Comment:
can you please change "nodeID" to "nodeName" for consistency with all other places?